apps/api/src/app/shared/services/in-memory-provider/redis-cluster-provider.ts

```diff
@@ -80,12 +80,14 @@ export const getRedisCluster = (): Cluster | undefined => {
   const { instances } = getRedisClusterProviderConfig();
 
   const options: ClusterOptions = {
+    enableOfflineQueue: false,
+    enableReadyCheck: true,
+    scaleReads: 'slave',
```

Comment:
This is the same configuration enabled for Elasticache Cluster provider.
`enableOfflineQueue=false` disables the ability of accumulate commands to the provider before being ready and therefore throwing an error if doing so

`enableReadyCheck=true` makes the client to return status `ready` only once is truly ready and not when the client has made connection with the server.

Reference: https://github.com/luin/ioredis#cluster